components/DataTable/src/Dependencies.props

```diff
@@ -17,12 +17,14 @@
         <PackageReference Include="CommunityToolkit.Uwp.Converters" Version="8.0.0-beta.1"/>
         <PackageReference Include="CommunityToolkit.Uwp.Extensions" Version="8.0.0-beta.1"/>
         <PackageReference Include="CommunityToolkit.Uwp.Controls.Sizers" Version="8.0.0-beta.2"/>
+        <PackageReference Include="CommunityToolkit.Uwp.Controls.HeaderedControls" Version="8.0.0-beta.1"/>
     </ItemGroup>
 
     <!-- WinUI 3 / WinAppSdk / Uno -->
     <ItemGroup Condition="'$(IsWinAppSdk)' == 'true' OR ('$(IsUno)' == 'true' AND '$(WinUIMajorVersion)' == '3')">
         <PackageReference Include="CommunityToolkit.WinUI.Converters" Version="8.0.0-beta.1"/>
         <PackageReference Include="CommunityToolkit.WinUI.Extensions" Version="8.0.0-beta.1"/>
         <PackageReference Include="CommunityToolkit.WinUI.Controls.Sizers" Version="8.0.0-beta.2"/>
+        <PackageReference Include="CommunityToolkit.WinUI.Controls.HeaderedControls" Version="8.0.0-beta.1"/>
```

Comment:
The HeaderedControls dependency is only needed for the samples now, eh? So, it can go in the Samples Dependencies file instead of the source for the control. `DataTable` doesn't specifically reference the Toolkit components since they're all based on either `ItemsPresenter` or the call out to `TreeView` itself.

Reply:
Seconded, if we only use HeaderedControls for the Sample, then we can move the PackageReference there.

Reply:
@niels9001 I made a commit for this.